Display/Documentation/HowToRunAnApplicationAtLaunchOnAnRPi.md

```diff
@@ -0,0 +1,14 @@
+How to Run an Application at Launch on a Raspberry Pi
+=====================================================
+
+Running an application at launch is fairly simple.  You simply use the `autostart` system to add to the boot procedure (there are other ways, but `autostart` guarantees that the X windowing system is loaded).  To do this, you need to create a file `/home/pi/.config/autostart/TetraVentilatorSplitter.desktop` (or whatever name `*.desktop`), and add this bit of code to it:
```

Comment:
I would remove use of the word _simple_ - since what's considered _simple_ is often too subjective, and remove the code snippet syntax for `autostart` - unless `autostart` is a command you're running on the pi.